.scalafmt.conf

```diff
@@ -0,0 +1,16 @@
+style = defaultWithAlign
```

Comment:
Do you think it makes a difference since most code is being touched anyways in such a code format PR? You guys better have no other open pull requests or branches you still plan to merge :).

Git blame has a mode to ignore whitespace btw.